astropath/chance.py

```diff
@@ -0,0 +1,99 @@
+""" Calculations related to P_chance """
+import numpy as np
+
+from scipy import interpolate
+
+from IPython import embed
+
+
+# Spline parameters(globals) are for rmag vs sigma
+driver_tck = (np.array([15., 15., 15., 15., 30., 30., 30., 30.]),
+       np.array([-6.41580144, -3.53188049, -1.68500105, -0.63090954, 0., 0., 0., 0.]), 3)
+driver_spl = interpolate.UnivariateSpline._from_tck(driver_tck)
+
+
+def driver_sigma(rmag):
+    """
+    Estimated incidence of galaxies per sq arcsec with r > rmag
+    using Driver et al. 2016 number counts.
+
+    Spline parameters (globals) are for rmag vs sigma
+
+    Args:
+        rmag (float or np.ndarray):
+
+    Returns:
+        float or np.ndarray:  Galaxy number density
+
+    """
+    return 10**driver_spl(rmag)
+
+
+def bloom_sigma(rmag):
+    """
+    Estimated incidence of galaxies per sq arcsec with r > rmag
+
+    Args:
+        rmag (float or np.ndarray):
```

Comment:
rmag (float or np.ndarray): r band magnitude of the galaxy

Reply:
yup